<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Arijit Panda - Tools</title>
    <link rel="stylesheet" href="style.css">
</head>
<body>

    <nav>
        <ul>
            <li><a href="index.html">About</a></li>
            <li><a href="research.html">Research</a></li>
            <li><a href="tools.html">Tools</a></li>
            <li><a href="publications.html">Publications</a></li>
            
        </ul>
    </nav>
    <main>
        <section id="tools">
            <h2>Tools</h2>
            <div class="tool">
                <h3>CNVpytor</h3>
                <p>CNVpytor is a tool for CNV detection and analysis from depth-of-coverage and B-allele frequency data. It provides a comprehensive set of features for visualizing, analyzing, and interpreting CNVs.</p>
                <p><a href="https://github.com/abyzovlab/CNVpytor" target="_blank">Learn more about CNVpytor</a></p>
            </div>
            <div class="tool">
                <h3>CNVpytor Track in igv.js</h3>
                <p>The track in igv.js that utilizes CNVpytor produced pytor file and whole genome variant files as input for on-the-fly read depth and BAF visualization, CNV/CNA calling and analysis. Embedding into HTML pages and Jupiter Notebooks enables convenient remote data access and visualization simplifying interpretation and analysis of omics data.
                </p>
                <p><a href="https://igv.org/doc/igvjs/#tracks/CNVPytor/" target="_blank">Track Info</a></p>
            </div>
            <div class="tool">
                <h3>GAL (Genome Annotator Light)</h3>
                <p>Genome Annotator Light (GAL) is a Docker-based package for genome analysis and data visualization. GAL integrates several existing tools and in-house programs inside a Docker Container for systematic analysis and visualization of genomes through a web browser. GAL takes various input types, ranging from raw Fasta files to fully annotated files, processes them through a standard annotation pipeline, and visualizes them on a web browser. Comparative genomic analysis is performed automatically within a given taxonomic class. GAL creates an interactive genome browser with clickable genomic feature tracks, a local BLAST-able database, a query page, and on-the-fly downstream data analysis using EMBOSS etc.</p>
                <p><a href="https://github.com/computational-genomics-lab/GAL" target="_blank">Learn more about GAL</a></p>
            </div>
        </section>
    </main>
    <footer>
        <p>&copy; 2025 Arijit Panda</p>
    </footer>
</body>
</html>
